src/bo4e/com/betrag.py

```diff
@@ -0,0 +1,45 @@
+"""
+Contains Betrag class
+and corresponding marshmallow schema for de-/serialization
+"""
+
+from decimal import Decimal
+
+import attr
+from marshmallow import fields, post_load
+from marshmallow_enum import EnumField  # type:ignore[import]
+
+from bo4e.com.com import COM, COMSchema
+
+# pylint: disable=too-few-public-methods
+from bo4e.enum.waehrungscode import Waehrungscode
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class Betrag(COM):
+    """
+    Die Komponente wird dazu verwendet, Summebeträge (beispielsweise in Angeboten und Rechnungen) als Geldbeträge a
+    bzubilden. Die Einheit ist dabei immer die Hauptwährung also Euro, Dollar etc…
+    """
+
+    # required attributes
+    wert: Decimal = attr.ib(validator=attr.validators.instance_of(Decimal))  #: Gibt den Betrag des Preises an.
+    waehrung: Waehrungscode = attr.ib(
```

Comment:
The BO4E docu writes `Waehrungs**C**ode` ?

Reply:
#236